index.css

```diff
@@ -0,0 +1,84 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+* {
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+/* Nav Section */
+
+#nav {
+  padding: 0 24px;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  height: 48px;
+  align-items: center;
+}
+
+.nav__logo {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  text-decoration: none;
+  color: #6070ff;
+}
+
+/* hero section */
+
+#hero {
+  background-image: url(images/hero_bg.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  padding: 0 24px;
+}
+
+.hero {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  height: 90vh;
+  justify-content: center;
+}
+
+.hero__heading {
+  width: 80%;
+  font-family: 'Poppins', sans-serif;
```

Comment:
- [Optional] I noticed you repeated the `font-family` property in multiple classes. You can consider creating a class for `font-family` and give the HTML elements the class name. For example `.poppins { font-family: 'Poppins', sans-serif; }` and then give the HTML elements the attribute `class="poppins"`. This makes your code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).